packages/emptystate/src/svg/bookmark-small.svg

```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64">
+  <g fill="none" fill-rule="evenodd">
```

Comment:
i'll look and see. these are exported and i just gave them a quick glance

Reply:
ok here are my findings:

- `fill` probably isn't needed - i'll remove it.
- `width` and `height` are stripped by svgr but i can just as well remove them as well.
- `stroke` some of the time is needed for the rounded ends - also i think it might be better for scaling and might be a smaller file size - but i don't really know. the alternative is to convert/flatten the paths(and other shapes) but i don't think that's going to gain us much

Reply:
ok i lied. `fill` with `fill-rule` is needed 😉